sources.list

```diff
@@ -0,0 +1,6 @@
+C:\Users\ta000\IdeaProjects\AlexeyGuliev\src\main\java\Main.java
```

Comment:
What does this file do? Why is it in the git?

Reply:
Forgot to delete (deleted)